src/cron/verification_remeber/verification_remember.task.ts

```diff
@@ -0,0 +1,84 @@
+import { getTicketsChannels } from "@/utils/chanels/chanels.utils";
+import { env } from "@/utils/env/env.util";
+import { nameof } from "@/utils/nameof.utils";
+import type { TaskResult, TaskType } from "arcscord";
+import { anyToError, error, ok, Task, TaskError } from "arcscord";
+import { subDays } from "date-fns";
+import { verificationKickEmbedBuilder } from "./kick_embed.builder";
+import { isUserHaveTicket } from "./verification_remember.helper";
+import { verificationWarnEmbedBuilder } from "./warn_embed.builder";
+
+export class VerificationRememberTask extends Task {
+
+  name = "Rappel de vérification";
+
+  type: TaskType = "cron";
+
+  interval = "0 0 4 * * *";
+
+  async run(): Promise<TaskResult> {
+    const guild = await this.client.guilds.fetch(env.SERVER_ID);
+    if (!guild) return error(
+      new TaskError({
+        message: "Unable to fetch Codeline Guild",
+        task: this,
+      })
+    );
+    const ticketChannels = await getTicketsChannels(
+      this.client
+    );
+    if (!ticketChannels) return error(
+      new TaskError({
+        message: `Unable to fetch ${nameof(ticketChannels)}`,
```

Comment:
Que retourne cette erreur ? Unable to fetch null ou Unable to fetch undefined non ? pas très utile...

Reply:
haha non dutout, nameof c'est une fonction très pratique que j'ai add et qui vien du c#. Sa retourne le nom de la variable que tu lui donne en string

Si ticketChannels est null | undefine ou tout ce que tu veut. nameof(ticketChannels) retourneras tjr "ticketChannels"

Donc cette erreur retourne -> Unable to fetch ticketChanels

Reply:
![CleanShot 2024-09-25 at 19 34 40@2x](https://github.com/user-attachments/assets/b32817f7-72a0-4371-bdac-85c364c2c482)
hmmm, sois je me suis louper, sois c'est pas vrm le cas